index.html

```diff
@@ -556,7 +556,7 @@ <h3 class="section-title">Contact Us</h3>
             Get in touch:
             <a href=""><i class="bi bi-twitter"></i></a>
             <a href=""><i class="bi bi-linkedin"></i></a>
-            <a href=""><i class="bi bi-github"></i></a> <br>
+            <a href="https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website"><i class="bi bi-github"></i></a> <br>
```

Comment:
```suggestion
            <a href="https://github.com/Open-Science-Community-Saudi-Arabia"><i class="bi bi-github"></i></a> <br>
```

Reply:
This is out [Twitter page](https://twitter.com/OpenSciSaudi) and [LinkedIn](https://www.linkedin.com/company/open-science-community-saudi-arabia/)

Reply:
THis is out means? it has been already fixed? or should I update that?  with Twitter and LinkedIn?  thanks